Adding an optional port parameter to the cassandra.yml

ryanhope2 opened the pull request:
- Add an optional port parameter to the cluster config
- defaulting port to 9042 if none provided.

BryanSD commented:
Looks like a reasonable PR. A couple of requests before merging:

- A good test for the new code. Should be as simple as adding a new environment (including the new `config` option) in [cassandra.yml](https://github.com/rackerlabs/cdeploy/blob/master/cdeploy/test/migrations/config/cassandra.yml) and writing a new [integration test](https://github.com/rackerlabs/cdeploy/blob/master/cdeploy/test/integration_test.py) (copying and tweaking the existing `DatabaseEnvironmentsTest` would probably be sufficient). I'm not sure we can change Cassandra's port on Travis CI, but even plugging in the default value into the test's config would be sufficient here so we know the code path is traveled.
- Spelling correction in your commit message 😉 

Thanks for the contribution!

ryanhope2 commented:
Hi Bryan,

I updated the PR, and it looks like it's passing Travis CI.  I updated the
Readme while I was revising the PR, hope that's ok.  Let me know if there's
anything else I can do to get this merged.

Thanks,
Ryan

On Mon, Jan 23, 2017 at 2:51 PM, Bryan Davidson <notifications@github.com>
wrote:

> Looks like a reasonable PR. A couple of requests before merging:
>
>    - A good test for the new code. Should be as simple as adding a new
>    environment (including the new config option) in cassandra.yml
>    <https://github.com/rackerlabs/cdeploy/blob/master/cdeploy/test/migrations/config/cassandra.yml>
>    and writing a new integration test
>    <https://github.com/rackerlabs/cdeploy/blob/master/cdeploy/test/integration_test.py>
>    (copying and tweaking the existing DatabaseEnvironmentsTest would
>    probably be sufficient). I'm not sure we can change Cassandra's port on
>    Travis CI, but even plugging in the default value into the test's config
>    would be sufficient here so we know the code path is traveled.
>    - Spelling correction in your commit message 😉
>
> Thanks for the contribution!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rackerlabs/cdeploy/pull/30#issuecomment-274597497>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB-I4AtxIBRw1iPQN7n8yU7hPLRe8_lTks5rVQTAgaJpZM4LrTGe>
> .
>



-- 

___

Ryan Hope

VP API and Architecture   //   *CoreDial, LLC*   //   coredial.com
//   [image:
Twitter] <https://twitter.com/coredial>   [image: LinkedIn]
<http://www.linkedin.com/company/99631>   [image: Google Plus]
<https://plus.google.com/104062177220750809525/posts>   [image: Blog]
<http://success.coredial.com/blog>
Hillcrest I, 751 Arbor Way, Suite 150, Blue Bell PA, 19422
*P: *215.297.4400 <2152974400> x534   //   *F: *215.297.4401 <2152974401>
   //   *E: *rhope@coredial.com

[image: Exceptional people. Proven Processes. Innovative Technology.
Discover CoreDial - watch our video]
<http://cta-redirect.hubspot.com/cta/redirect/210539/4c492538-6e4b-445e-9480-bef676787085>

The information transmitted is intended only for the person or entity to
which it is addressed and may contain confidential and/or privileged
material. Any review, retransmission,  dissemination or other use of, or
taking of any action in reliance upon, this information by persons or
entities other than the intended recipient is prohibited. If you received
this in error, please contact the sender and delete the material from any
computer.
